net/uradvd/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uradvd
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/freifunk-gluon/uradvd.git
+PKG_SOURCE_DATE:=2021-09-14
+PKG_SOURCE_VERSION:=9b0da60e27c67305d251b10163e388191d566d7a
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uradvd
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=A tiny radvd
+endef
+
+define Package/uradvd/description
+	Advertise an IPv6 prefix/route via SLAAC.
+endef
+
+define Package/uradvd/install
+	$(INSTALL_DIR) $(1)/etc/config/
+	$(INSTALL_DATA) ./files/etc/config/uradvd $(1)/etc/config/
+	$(INSTALL_DIR) $(1)/etc/init.d/
+	$(INSTALL_DATA) ./files/etc/init.d/uradvd $(1)/etc/init.d/
```

Comment:
The initscript needs to be installed using `INSTALL_BIN`, so it is made executable.